src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -624,7 +624,7 @@ private function registerProfilerConfiguration(array $config, ContainerBuilder $
     private function registerWorkflowConfiguration(array $config, ContainerBuilder $container, XmlFileLoader $loader)
     {
         if (!$config['enabled']) {
-            if (!class_exists(Workflow\Workflow::class)) {
+            if ($container->hasDefinition(WorkflowDumpCommand::class)) {
```

Comment:
the "if" can be removed: removeDefinition already does it internally

Reply:
Correct. Thks @nicolas-grekas !